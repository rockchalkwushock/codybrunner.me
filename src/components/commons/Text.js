import React from 'react'
import styled from 'styled-components'

const StyledText = styled.p`
  color: ${({ alt, theme }) =>
    alt ? theme.colors.secondary : theme.colors.alt};
  margin: unset;
  padding: 1rem 0;

  @media (min-width: ${({ theme }) => theme.screen.large}) {
    padding: 2rem;
    width: 50%;
  }
`

const Text = ({ alt, children, text }) => (
  <StyledText alt={alt}>{text || children}</StyledText>
)

export default Text