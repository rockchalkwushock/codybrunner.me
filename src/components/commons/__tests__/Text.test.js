import React from 'react'
import { shallow } from 'enzyme'

import Text from '../Text'

describe('Component: <Text />', () => {
  let wrapper
  beforeEach(() => {
    wrapper = shallow(<Text />)
  })

  test('should render correctly', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
