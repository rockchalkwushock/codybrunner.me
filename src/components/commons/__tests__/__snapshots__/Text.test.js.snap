// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: <Text /> should render correctly 1`] = `
<p
  className=""
/>
`;
